nilearn/decoding/tests/test_objective_functions.py

```diff
@@ -79,7 +78,7 @@ def test_logistic_loss_derivative(n_samples=4, n_features=10, decimal=5):
         decimal=decimal,
     )
 
-    np.testing.assert_almost_equal(
+    assert_almost_equal(
```

Comment:
I don't understand why this is checking exactly the same thing as previously.. 🤔

Reply:
As far as I go back in the git blame, this has always been there and I don't understand why.

https://github.com/nilearn/nilearn/blame/f8a028e31af73074df5830110c5ea625168405a9/nilearn/decoding/tests/test_objective_functions.py#L70

Possible explanation: no one noticed until now.

Reply:
I think this possible explanation is indeed correct. 😅 
I would vote for removing one of the two assertions.

Reply:
ok will remove it